frontend/src/app/features/work-packages/routing/wp-view-page/wp-view-page.component.ts

```diff
@@ -102,6 +102,9 @@ export class WorkPackageViewPageComponent extends PartitionedQuerySpacePageCompo
 
   ngOnInit() {
     super.ngOnInit();
+    this.wpTableFilters.hidden.push(
```

Comment:
Hi @b12f If we add 'Project' to hidden list in wp-view-filters.service.ts, it will be hidden from filters everywhere like form configuration or boards filters, where we have not added 'Include projects' modal yet.

Reply:
Yes, correct, we can't push it to the filters service, as for example in the table configuration modal, that filter should still be visible